frida_tools/repl.py

```diff
@@ -197,13 +195,16 @@ def _stop(self):
             if not self._quiet:
                 self._print("\nThank you for using Frida!")
 
+        def _get_script_name(self, path):
+            return os.path.splitext(os.path.basename(path))[0]
```

Comment:
This should go below the code that is using it. (Newspaper code structure.)

Reply:
Fixed and rebased against main